### DISCUSSION TITLE
up to date on app store?

### DISCUSSION THREAD

**Discussion by momonala** [aaronpk/Overland-iOS#112] [Q&A] [answered]
Hi there, thanks for this great tool! 

I noticed that the version of the app on Apple's app store is from 2018, where as there are several more recent commits here. Is there a plan to bump the app store's version?

**Comment by aaronpk** [comment, marked as answer]
I was originally planning on getting a few more features into the [next app store version](https://github.com/aaronpk/Overland-iOS/projects/1), but development has slowed down significantly and at this point I should just update with the current code.

**Comment by momonala** [comment]
Good to know, thanks!

**Comment by groupnet** [comment]
Would be great to see the actual code on the AppStore! Thanks!

**Comment by aaronpk** [comment]
I've got a version prepared for the next release, it's waiting approval for TestFlight testing now! If you are interested in testing the pre-release version and providing feedback, let me know here!

https://forms.gle/8HvoPsMFgTwN4uup8

**Reply by momonala** [reply]
Awesome! Thanks for the changes, looking forward to it. Id be happy to help test it out.

**Comment by aaronpk** [comment]
Version 1.3 is now available in the App Store! New features include:

• Updates for new iOS 16/17 location APIs to enable continuously running in the background
• Added new settings to reduce the amount of data collected, either by distance or by time
• Home Assistant compatibility
• Configure different settings for background tracking and tracking during trips
• Dark mode!
• Step count is included in trip records
• Added additional settings: access token, unique device ID
• Optionally keep the screen from going to sleep during a trip
• Configure settings from the response from your server
